modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/checkpoint/Checkpointer.java

```diff
@@ -340,6 +346,14 @@ void doCheckpoint() throws IgniteInternalCheckedException {
                     }
                 }
 
+                try {
+                    logSyncer.sync();
+                } catch (Exception e) {
+                    log.error("Failed to sync write-ahead log during checkpoint processing ", e);
```

Comment:
```suggestion
                    log.error("Failed to sync write-ahead log during checkpoint", e);
```

Reply:
I don't remember whether we should end the statement with a dot or not, please check other messages

Reply:
In most messages there is no dot